pgvectorscale/Cargo.toml

```diff
@@ -11,12 +11,13 @@ name = "pgrx_embed_vectorscale"
 path = "./src/bin/pgrx_embed.rs"
 
 [features]
-default = ["pg17"]
+default = ["pg18"]
```

Comment:
We can also drop support for pg13 and pg14 at this point.  Let's go ahead and do that, to tidy things up a bit.